### PR TITLE
Fixed issue that campaign banner was covering the map

### PR DESCRIPTION
**Pull request by yashha**
Also added an hide button
image
image
#510

### CODE REVIEW

File: app/components/app/Header/Header.css [hunk 1]
```diff
@@ -116,10 +116,25 @@ header {
...
     text-align: center;
+    z-index: 50;
 }
 .campaign-button {
     margin-left: 50px;
+    margin-right: 10px;
+}
+.campaign-close {
+    cursor: pointer;
+    cursor: hand;
```
Comment (sacdallago) [inline app/components/app/Header/Header.css]: Cursor `pointer` or cursor `hand`? It's not necessary to use both anymore: http://stackoverflow.com/a/3088012/5708679

**Comment by sacdallago** [comment]
Looks good, you might want to change that 2 CSS lines and compress them into one, then it's merge-ready

**Comment by sacdallago** [comment]
:D now this branch has conflicts 😭 sorry

**Comment by yashha** [comment]
I think it can be merged.
If we wait longer, I could really saved my time for other things ;)

**Pull request merged**